model/project.rb

```diff
@@ -73,4 +73,5 @@ def self.feature_flag(*flags)
   end
 
   feature_flag :enable_postgres
+  feature_flag :storage_policy
```

Comment:
Our model is "GithubStoragePolicy", we may use `github_storage_policy` as the feature flag name.

Reply:
you can do;
```ruby
feature_flag :enable_postgres, :github_storage_policy
```